src/Sql/Sql/Database Backup/Cmdlet/RemoveAzureRmSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -107,6 +108,16 @@ public class RemoveAzureRmSqlDatabaseLongTermRetentionBackup : AzureSqlDatabaseL
         [ValidateNotNullOrEmpty]
         public string BackupName { get; set; }
 
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ParameterSetName = RemoveBackupDefaultSet,
+            Position = 0,
```

Comment:
optional parameters should not have positions and definitely should not be position  0